enarx-keep-sgx-shim/src/heap.rs

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use core::mem::size_of;
+use core::slice::from_raw_parts_mut as slice;
+
+use nolibc::x86_64::error::Number as ErrNo;
+use page::Page;
+use span::{Line, Span};
+
+type Word = usize;
```

Comment:
I wonder if just plain `u64` is more clear here.

Reply:
I'd prefer to move all our `u64`s to `usize`s. But `StateSaveArea` is the biggest problem. We probably need to generate getters/setters for `usize`.

Reply:
This should be addressed by debt bug #424.